x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/type/SqlDataTypes.java

```diff
@@ -274,7 +274,6 @@ public static boolean isFromDocValuesOnly(DataType dataType) {
                 || dataType == SCALED_FLOAT // because of scaling_factor
                 || dataType == CONSTANT_KEYWORD
                 || dataType == GEO_POINT
-                || dataType == GEO_SHAPE
```

Comment:
@elastic/es-ql does anyone know why the `geo_shape` field was here as `isFromDocValuesOnly` when the field did not have doc values?